### ISSUE TITLE
Calculation don't appear to work with a group of fields

### ISSUE THREAD

**Issue by richardgrantham26** [verbb/formie#1214]
### Describe the bug

I have setup a group of number fields, and a calculation field to total these values. When it is within a group it doesn't seem to total.

### Steps to reproduce

1. Add a group field
2. Add 2 number fields
3. Add a calculation field to total these values

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.3.2.1

### Plugin version

2.0.19

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.19",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.20](https://github.com/verbb/formie/releases/tag/2.0.20)
